packages/google_maps_flutter/google_maps_flutter/lib/src/controller.dart

```diff
@@ -18,6 +18,35 @@ class GoogleMapController {
   /// The mapId for this controller
   final int mapId;
 
+  /// State map of the stream subscriptions for this controller, used internally.
```

Comment:
Internal implementation details should not be public properties; this would make it impossible for us to ever change without a major version bump.